ironfish/src/wallet/errors.ts

```diff
@@ -2,13 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+import { CurrencyUtils } from "../utils"
+
 export class NotEnoughFundsError extends Error {
   name = this.constructor.name
 
   constructor(assetId: Buffer, amount: bigint, amountNeeded: bigint) {
     super()
-    this.message = `Insufficient funds: Needed ${amountNeeded.toString()} but have ${amount.toString()} for asset '${assetId.toString(
-      'hex',
-    )}'`
+    this.message = `Insufficient funds: Needed ${CurrencyUtils.renderIron(amountNeeded.toString())} but have '${CurrencyUtils.renderIron(amount, true, assetId.toString('hex',))}'`
```

Comment:
There is a sneaky extra comma in this toString()

Reply:
Also we don't need to do amountNeeded.toString() here since `renderIron` takes either ore as bigint or encoded ore as a string.

Reply:
`assetId.toString('hex',)` becomes `assetId.toString('hex')